TLS_E2E_Automation/cypress/support/tlshealthcheckcommands.js

```diff
@@ -271,9 +272,9 @@ Cypress.Commands.add( 'validateTlsBuyPage', () => {
 	cy.acceptCookieBanner();
 	cy.get(subscribeButton).click();
 	cy.get(printAndDigitalPrice,{ timeout: 5000 }).should('be.visible');
-	cy.get(printPrice).should('be.visible');
-	cy.get(DigitalPrice).should('be.visible');
```

Comment:
Can you make it as div[class='product  print' ] > div[class=price], div[class*='print-digital'] > div[class='price'] and div[class*='product  digital'] > div[class='price'] instead of price.eq(0),price.eq(1),price.eq(2)

Reply:
done @balajikrishnannewsuk